screens/html/product-list.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./fontawsome/all.min.css">
+    <link rel="stylesheet" href="./fontawsome/fontawesome.min.css">
+    <link
+    rel="stylesheet"
+    href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+    crossorigin="anonymous"
+  />
+    <link rel="stylesheet" href="https://dreamy-wright-6bb3af.netlify.app/components/components.css">
+
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <nav class="navbar flex-space-btw wd-100">
+            <div class="nav-logo"><i class="fas fa-jedi nav-logo-icon"></i> <span class="nav-logo-txt">Jedi Store</span>
+            </div>
+            <div class="navbar-search">
+              <button class="search-btn"><i class="fas fa-search"></i></button>
+              <input class="search-input" type="text" name="search" placeholder="Search Here....">
+            </div>
+            <div class="navbar-links flex-center">
+              <a class="nav-login-link" href="./screens/html/login.html">
+                <button class="nav-login-btn">Login</button>
+              </a>
+              <ul class="nav-list">
+                <li class="nav-list-item">
+                  <a class="nav-list-link" href="#"><i class="far fa-heart nav-link-icon"></i></a>
+                </li>
+                <li class="nav-list-item">
+                  <a class="nav-list-link icon-badge pos-rel " href="#"><i class="fas fa-cart-plus nav-link-icon"></i><div class="badge-number flex-center badge-on-icon pos-abt">4</div></a>
```

Comment:
badge icon can be primary or cta color just suggestion as blue contrasts whole styling

Reply:
done